site/docs/src/json/jwt/refresh-token-get-response.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Can you format this file? I think our default formatting rules will bring this curly brace up to the previous line

Reply:
I pulled the curly brace up. I don't have intellij so can't use the formatting, unfortunately.